jest.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  preset: "ts-jest/presets/js-with-ts",
+  testMatch: [
+    "<rootDir>/packages/*/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
+    "<rootDir>/packages/*/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}",
+  ],
+  coveragePathIgnorePatterns: [
+    "/node_modules/",
+    "dist/",
```

Comment:
Would we need to ignore dist also from within the packages?

Reply:
i believe this will ignore any instance of `dist`, although i might be wrong. we're not using the coverage stats yet though, so it probably doesn't matter?